tests/e2e/cucumber/features/smoke/admin-settings/spaces.ocis.feature

```diff
@@ -122,3 +122,46 @@ Feature: spaces management
       | David | viewer  |
       | Edith | viewer  |
     And "Alice" logs out
+
+
+  Scenario: admin user can manage the spaces created by other space admin user
+    Given "Admin" creates following users using API
+      | id    |
+      | Alice |
+      | Brian |
+      | Carol |
+    And "Admin" assigns following roles to the users using API
+      | id    | role        |
+      | Alice | Admin       |
+      | Brian | Space Admin |
+      | Carol | Space Admin |
+    When "Brian" logs in
+    And "Brian" creates the following project spaces using API
+      | name   | id     |
+      | team A | team.a |
+    When "Carol" logs in
+    And "Carol" creates the following project spaces using API
+      | name   | id     |
+      | team B | team.b |
+    When "Alice" logs in
+    And "Alice" opens the "admin-settings" app
+    And "Alice" navigates to the project spaces management page
+    When "Alice" updates the space "team.a" name to "brian team" using the context-menu
+    And "Alice" updates the space "team.b" name to "carol team" using the context-menu
+    And "Alice" updates the space "team.a" quota to "10" using the context-menu
+    And "Alice" updates the space "team.b" quota to "50" using the context-menu
+    And "Alice" disables the space "team.a" using the context-menu
+    And "Alice" disables the space "team.b" using the context-menu
+    And "Alice" enables the space "team.a" using the context-menu
+    And "Alice" enables the space "team.b" using the context-menu
+    And "Alice" disables the space "team.a" using the context-menu
+    And "Alice" disables the space "team.b" using the context-menu
+    And "Alice" deletes the space "team.a" using the context-menu
+    And "Alice" deletes the space "team.b" using the context-menu
+    Then "Alice" should not see the following spaces
+      | id     |
+      | team.a |
+      | team.b |
+    And "Alice" logs out
+    And "Brian" logs out
+    And "Carol" logs out
```

Comment:
```suggestion
    And "Carol" logs out
    
    
    Scenario: admin user can manage all spaces in the admin settings via the batch actions
    Given soon .......

```

Reply:
i thought of but @saw-jan suggested one is enough.

Reply:
Should be covered by other scenarios. But @SagarGi if you like to then you can combine both ways in a single scenario.
Like swikriti suggested here https://github.com/owncloud/web/pull/9132/files#r1212796752
disables both using batch action, enable using context menu and so on
I don't think we need separate scenarios

Reply:
ok then some test code refactor needs to be done inorder to make it work (combine both).

Reply:
Can be covered only through the batch actions. Confirmed from viktor.